Create install.sh

joseto1298 opened the pull request.

joseto1298 committed:
Create install.sh

bofh69 commented:
This looks awesome!

I'm away on vacation without a computer right now so I'll review it when I come back home.

Please add spdx copyright tags to the file, like this; 
```sh
# SPDX-FileCopyrightText: [year] [copyright holder] <[contact address]>
```
Where FileCopyrightText is the SPDX identifier of the license.

joseto1298 committed:
ADD CopyrightText

bofh69 commented:
Oh, one more thing. Please change the mode of the script to be runable: chmod 755 install.sh

joseto1298 committed:
Add interactive spoolman2slicer setup script

- Bash script to install/update spoolman2slicer and its Python dependencies
- Copies slicer templates to user config (~/.config/spoolman2slicer)
- Generates filament configuration files for OrcaSlicer, PrusaSlicer, or SuperSlicer
- Saves user preferences (slicer, output folder, Spoolman URL)
- Includes interactive menu with validation and error handling
